requirements.txt

```diff
@@ -0,0 +1,6 @@
+virtualenv
```

Comment:
Please test the heroku deployment without virtualenv and if it works properly
- remove `virtualenv` from requirementx.txt.
- update the `ON_HEROKU` check in `generate.sh` by removing the creation of `virtualenv` from it and place it in a new section which should only occur if `WEBUI` is not `true`